UnityClient/Assets/Scripts/TowerDefence/GameManager.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GameManager : MonoBehaviour
+{
+
+    public static bool GameIsOver;
+
+    //public GameObject gameOverUI;
+
+    public GameObject completeLevelUI;
+
+    void start()
+    {
+        GameIsOver = false;
+    }
+
+    void Update()
```

Comment:
This isn't doing anything other than providing a performance hit. Is this even needed?